### DISCUSSION TITLE
Changing the size/color of multi-select dot?

### DISCUSSION THREAD

**Discussion by erwin** [davatorium/rofi#1671] [Theming]
Is there a way to customize the _multi-select_ dot that's activated with <kbd>shift</kbd> + <kbd>enter</kbd> in `rofi -dmenu -multi-select` mode?

![image](https://user-images.githubusercontent.com/11534217/183279829-37669d00-b2b6-4f57-bc12-296e028a733d.png)

I've checked the Rofi Theme man page, searched fairly extensively on Reddit and Google, and the github Issues list, but no luck so far.

These are the dots I would like to change. For my eyes, this is not nearly enough contrast, and ideally I could also make the dots much bigger.

I also tried to switch to all of the other themes in /usr/share/rofi/themes, but none seem to have a different appearance for the `selected` bullet.

![image](https://user-images.githubusercontent.com/11534217/183280257-0d714436-2828-49ea-971d-d39c94f842c4.png)

The closest I've found is this Reddit question, but that was about the possibility of theme elements being overlaid on top of the dots, rather than how to directly change the appearance of the dots.

https://www.reddit.com/r/qtools/comments/k9oy99/customize_multiselect_mode_in_rasi/

**Comment by DaveDavenport** [comment]
the multi-selection dots are a huge hack. They have been removed for the next release.

**Comment by erwin** [comment]
So in future releases how will you know WHICH row is selected when you do multi select?

Can you change the COLOR of the multi-select items?

**Reply by DaveDavenport** [reply]
https://github.com/davatorium/rofi/blob/next/doc/rofi-dmenu.5.markdown
see the `-ballot-selected-str` and `-ballot-unselected-str` entry.

**Reply by erwin** [reply]
Wow! That's looking really cool!

Here I was getting grumpy and entitled about lack of ability to tweak the `bullet` formatting, and you've already go some actual checkboxes for multi select mode!

![image](https://user-images.githubusercontent.com/11534217/183280977-2356c00c-48bb-4e0c-8146-d2542e9edc72.png)
